### DISCUSSION TITLE
Is it possible that vLLM triton backend be implemented in none-decoupled mode?

### DISCUSSION THREAD

**Discussion by gaoxt1983** [vllm-project/vllm#3163] [General]
Sorry I'm not an English speaker, so forgive my poor English.
I want to use Triton as our model inference server and vLLM as a backend. But since vLLM's triton backend only support decoupled mode. Is it possible to implement none-decoupled mode myself? Is there anything I should be aware of?

**Comment by simon-mo** [comment]
can you explain what decoupled mode mean? also feel free to write the question in Chinese.

**Reply by gaoxt1983** [reply]
Thanks. 我想使用vLLM来做为triton的backend。但我们想使用非decoupled模式（段式请求）。想请问在triton实现vLLM的段式传输，是否可行？有什么注意的地方吗？
